app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -13,3 +13,12 @@ attributes :sockets, :cores, :ram
 attributes :instance_multiplier, :stacking_id, :multi_entitlement
 attributes :type
 attributes :name => :product_name
+attributes :unmapped_guest
+attributes :virt_only
```

Comment:
haha I wasn't sure if you had an opinion either way :) My opinion is to leave virt_only in because it is more explicit and can be used in logic easier when people use the api or in UI/hammer. Let me know if I am off base there.